### PR TITLE
[lambda-log_v2.x.x] Add definitions

### PR DESCRIPTION
**Pull request by joshuanapoli**
- Links to documentation: https://github.com/KyleRoss/node-lambda-log#readme
- Link to GitHub or NPM: https://github.com/KyleRoss/node-lambda-log#readme
- Type of contribution: new definition

### CODE REVIEW

File: definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    replacer?: (key: string, value: any) => any;
+    stdoutStream?: WriteStream;
+    stderrStream?: WriteStream;
+  }
+  declare interface LogLevels {
+    info: "info";
+    warn: "warn";
+    error: "error";
+    debug: "log" | false;
+    [key: string]: any;
```
Comment (pascalduez) [inline definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js:46]: What's that supposed to do?

File: definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  }
+  declare export class LambdaLog mixins EventEmitter {
+    LambdaLog: LambdaLog;
+    options: LambdaLogOptions;
+    console: Console;
+  [key: string]: any;
```
Comment (pascalduez) [inline definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js:52]: Same question here.

File: definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    constructor(options?: LambdaLogOptions, levels?: any): this;
+    log(level: string, msg: string, meta?: { ... }, tags?: string[]): LogMessage;
```
Comment (pascalduez) [inline definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js:54]: `level` could be typed more precisely as far as I can see.

File: definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js (hunk 1 above)
Comment (joshuanapoli) [reply definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js]: The library allows custom log levels... I changed to statically typed custom log levels in fabdcfa.

File: definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js (hunk 2 above)
Comment (joshuanapoli) [reply definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js]: The library allows custom log levels... I changed to statically typed custom log levels in fabdcfa.

File: definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js (hunk 3 above)
Comment (joshuanapoli) [reply definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js]: OK. I changed to statically typed custom log levels in fabdcfa.

File: definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    on(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    once(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    prependListener(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    prependOnceListener(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    removeListener(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    off(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    removeAllListeners(event?: string | symbol): LambdaLog<LogLevels>,
+    setMaxListeners(n: number): LambdaLog<LogLevels>,
+    getMaxListeners(): number,
+    listeners(event: string | symbol): Function[],
```
Comment (pascalduez) [inline definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js:73]: `Function` is deprecated, and is equal to `any`. Would be nice if you could type it a bit more precisely.

**Review by pascalduez — CHANGES_REQUESTED** [review]
Thanks! 
Just two `Function` to change and we are goof to go ;)

File: definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js (hunk 4 above)
Comment (joshuanapoli) [reply definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js:72]: Thanks. I replaced the deprecated `Function` keyword and tightened the related types.

**Comment by joshuanapoli** [comment]
Sorry, but my latest commit 71d85de isn't showing up in this PR. I think that this is a problem within GitHub. I'll re-create this PR if it doesn't show up in the next day.

**Review by pascalduez — APPROVED** [review]
Thanks!

**Pull request merged**